message_sender/factory.py

```diff
@@ -339,6 +341,7 @@ def send_hsm(self, whatsapp_id, content):
             urllib_parse.urljoin(self.api_url, "/v1/messages"),
             json={
                 "to": whatsapp_id,
+                "ttl": 604800,  # 7 days
```

Comment:
I don't think that we want this to be static, I think we want it to reflect the `MESSAGE_TTL` in the channel configuration.

Reply:
I think by default, whatsapp sets the TTL to 7 days. So if `MESSAGE_TTL` is not set, it will be `None`, so in the case of `None`, we don't want to set it here, and if it's not `None`, then we do want to set it.